CHANGES.md

```diff
@@ -84,6 +84,7 @@
   * Transform service controller container.
   * Spark3 job server container.
 * Added support for batched writes to AWS SQS for improved throughput (Java, AWS 2).([#21429](https://github.com/apache/beam/issues/21429))
+* Added support to run `mypy` on user pipelines ([#27906](https://github.com/apache/beam/issues/27906))
```

Comment:
this change didn't make a release cut unfortunately.

Reply:
It's currently under 2.50.0 Unreleased, I don't see a 2.51 so what should we do with this?

Reply:
If you pull in the changes from the master branch there is a 2.51 section - https://github.com/apache/beam/blob/master/CHANGES.md